src/guides/v2.2/rest/tutorials/orders/order-create-order.md

```diff
@@ -1573,6 +1573,19 @@ Not applicable
 
 {% endcollapsible %}
 
+### Send payment information for guest customer{#send-payment}
+
+**Endpoint:**
+
+`POST <host>/rest/<store_code>/V1/guest-carts/<cartId>/payment-information`
+
+**Headers:**
+
+`Content-Type` `application/json`
+
+`Authorization` `Bearer <customer token>`
```

Comment:
Do we need token for a guest user? I believe the masked quote ID should do the trick, but I might be wrong.

Reply:
@rogyar Yes no need of token I'll update the changes

Reply:
Updated